infrastructure/ai/prompt_template_loader.py

```
@@ -0,0 +1,307 @@
+                f"模板加载失败: node={node_name}, role={role}, "
+                f"template={template_name}: {e}"
+            ) from e
+
+    def render_with_fallback(
+        self,
+        node_name: str,
+        role: str,
+        fallback: str,
+        **variables: Any,
+    ) -> str:
+        """渲染模板，文件缺失时使用 fallback 字符串。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)